data/product.attribute.csv

```diff
@@ -1,14 +1,8 @@
 id,name,create_variant
-product_attribute_BbA,Baby by Age,TRUE
 product_attribute_BS,Bra Sizes,TRUE
-product_attribute_CbA,Children by Age,TRUE
 product_attribute_DS,Diaper Sizes,TRUE
-product_attribute_MS,Mixed Sizes,FALSE
-product_attribute_O,Onesize,FALSE
 product_attribute_CS,Clothing Sizes,TRUE
-product_attribute_CSS,Children Shoe Sizes,TRUE
-product_attribute_FSS,Female Shoe Sizes,TRUE
-product_attribute_MSS,Male Shoe Sizes,TRUE
+product_attribute_SS, Shoe Sizes,TRUE
 product_attribute_G,Gender,TRUE
 product_attribute_AG,Age_Group,TRUE
```

Comment:
@naphets123 hope you don't mind me jumping in, but is there a reason for keeping the underscore in Age group vs the others?

Reply:
+1